.github/workflows/lint.yml

```diff
@@ -56,6 +56,33 @@ jobs:
         export ADDITIONAL_LINTRUNNER_ARGS="--skip CLANGTIDY,CLANGFORMAT"
         .github/scripts/lintrunner.sh
 
+  # NB: This doesn't work with forked PR https://github.com/parkerbxyz/suggest-changes/issues/33 :(
+  lintrunner-autoformat:
```

Comment:
Let's make this run only when lint fails.  Hopefully, people won't need it most of the time

Reply:
Also, I don't think the review can be blank https://github.com/pytorch/pytorch/actions/runs/10967111239/job/30456284103?pr=136382#step:5:220, and passing lint means an empty review